CMakeLists.txt

```diff
@@ -12,6 +12,6 @@ add_subdirectory(subsys)
 add_subdirectory(drivers)
 
 # Add the mandatory nrfxlib repository
-set(               NRFXLIB_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../nrfxlib)
+set(               NRFXLIB_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../nrfxlib CACHE INTERNAL "")
```

Comment:
What is the effect of this change?

Reply:
This caches a value of the NRFXLIB_DIR variable globally so that the variable could be used later outside of the folder where it was set.